xline/src/main.rs

```diff
@@ -110,17 +110,26 @@
     clippy::multiple_crate_versions, // caused by the dependency, can't be fixed
 )]
 
-use std::{collections::HashMap, path::PathBuf};
-
-use anyhow::Result;
+use anyhow::{anyhow, Result};
 use clap::Parser;
 use jsonwebtoken::{DecodingKey, EncodingKey};
 use opentelemetry::{global, runtime::Tokio, sdk::propagation::TraceContextPropagator};
 use opentelemetry_contrib::trace::exporter::jaeger_json::JaegerJsonExporter;
+use std::{collections::HashMap, env, path::PathBuf};
```

Comment:
use std before use third-party crates for same code style

Reply:
I suggest use the nightly version of `rustfmt` which has these options that can automatically organize imports.
```toml
imports_granularity = "Crate"
group_imports = "StdExternalCrate"
```

Reply:
Done